<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reservation</class>
 <widget class="QDialog" name="Reservation">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Réservation</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>621</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabAjout">
    <attribute name="title">
     <string>Ajouter une réservation</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditP">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Prix en Dinard</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelT">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Trajet</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNP">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Nombre de place</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>l'adresse Email</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDHD">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Dates de départs</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEditAE">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelDH">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Date et heure de depart</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditNB">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;Numéro de bus&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTD">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Liste des stations de départ</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelNR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Numéro de la réservation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonValider">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxNP">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTA">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Liste des stations d'arrivées</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxDHA">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>100</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Dates d'arrivées</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="spinBoxNum">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoad">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>0</y>
       <width>691</width>
       <height>481</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPDF">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPBus">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>lineEditP</zorder>
    <zorder>labelP</zorder>
    <zorder>labelT</zorder>
    <zorder>labelNP</zorder>
    <zorder>label</zorder>
    <zorder>comboBoxDHD</zorder>
    <zorder>lineEditAE</zorder>
    <zorder>labelDH</zorder>
    <zorder>textEditNB</zorder>
    <zorder>comboBoxTD</zorder>
    <zorder>labelNR</zorder>
    <zorder>pushButtonValider</zorder>
    <zorder>spinBoxNP</zorder>
    <zorder>comboBoxTA</zorder>
    <zorder>comboBoxDHA</zorder>
    <zorder>spinBoxNum</zorder>
    <zorder>pushButtonLoad</zorder>
    <zorder>pushButtonPDF</zorder>
    <zorder>lineEditPBus</zorder>
   </widget>
   <widget class="QWidget" name="tabModifier">
    <attribute name="title">
     <string>Modifier une Réservation</string>
    </attribute>
    <widget class="QLabel" name="labelRechercheModif">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Veuillez entrer le numéro de la réservation</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditReModif">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReModif">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelNPModif">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Nombre de place</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxNPModif">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTModif">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Trajet</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDHModif">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Date et heure de depart</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelP_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Prix en Dinard</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPModif">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelAModif">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>l'adresse Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAEModif">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>290</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRechModif">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-70</x>
       <y>-50</y>
       <width>701</width>
       <height>581</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTDModif">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Liste des stations de départ</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxTAModif">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Liste des stations d'arrivées</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxDHDModif">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Dates de départs</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxDHAModif">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>170</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Dates d'arrivées</string>
      </property>
     </item>
    </widget>
    <zorder>label_3</zorder>
    <zorder>labelRechercheModif</zorder>
    <zorder>lineEditReModif</zorder>
    <zorder>pushButtonReModif</zorder>
    <zorder>labelNPModif</zorder>
    <zorder>spinBoxNPModif</zorder>
    <zorder>labelTModif</zorder>
    <zorder>labelDHModif</zorder>
    <zorder>labelP_2</zorder>
    <zorder>lineEditPModif</zorder>
    <zorder>labelAModif</zorder>
    <zorder>lineEditAEModif</zorder>
    <zorder>pushButtonRechModif</zorder>
    <zorder>comboBoxTDModif</zorder>
    <zorder>comboBoxTAModif</zorder>
    <zorder>comboBoxDHDModif</zorder>
    <zorder>comboBoxDHAModif</zorder>
   </widget>
   <widget class="QWidget" name="tabSupp">
    <attribute name="title">
     <string>Supprimer une Réservation</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditReSupp">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelRechercheSupp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Veuillez entrer le numéro de la réservation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReSupp">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-30</y>
       <width>611</width>
       <height>561</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>lineEditReSupp</zorder>
    <zorder>labelRechercheSupp</zorder>
    <zorder>pushButtonReSupp</zorder>
   </widget>
   <widget class="QWidget" name="tabConsulter">
    <attribute name="title">
     <string>Consulter une Réservation</string>
    </attribute>
    <widget class="QTableView" name="tabReservation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTri">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>390</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string> tri selon le num de réservation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-30</y>
       <width>621</width>
       <height>561</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPDF_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>390</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <zorder>label_5</zorder>
    <zorder>tabReservation</zorder>
    <zorder>pushButtonTri</zorder>
    <zorder>pushButtonPDF_2</zorder>
   </widget>
   <widget class="QWidget" name="tabRecherche">
    <attribute name="title">
     <string>Recherche</string>
    </attribute>
    <widget class="QTableView" name="tabRecherche_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>581</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelRechercheSupp_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Veuillez entrer le numéro de l'Identifiant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRecherche">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReOk">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>611</width>
       <height>461</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_6</zorder>
    <zorder>tabRecherche_2</zorder>
    <zorder>labelRechercheSupp_2</zorder>
    <zorder>lineEditRecherche</zorder>
    <zorder>pushButtonReOk</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
